resources/cms/charts/cms-asyncapi-service/templates/deployment.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  template:
+    metadata:
+      annotations:
+        sidecar.istio.io/inject: "false"
+      labels:
+        app: {{ template "name" . }}
+        release: {{ .Release.Name }}
+    spec:
+      strategy:
+        type: RollingUpdate
+        rollingUpdate:
+          maxUnavailable: 0
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.global.containerRegistry.path }}/{{ .Values.global.cmsAsyncAPIService.dir }}{{ .Values.image.name }}:{{ .Values.global.cmsAsyncAPIService.version }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          env:
+            - name: APP_VERBOSE
+              value: "true"
+            - name: APP_SERVICE_PORT
+              value: "{{ .Values.service.internalPort }}"
+            - name: APP_SERVICE_HOST
+              value: "0.0.0.0"
```

Comment:
move to values.yaml

Reply:
I disagree, this value shouldn't be exposed in `values.yaml`